docs/src/code/style-guide.adoc

```diff
@@ -400,79 +399,226 @@ and longer documents may also chose to set
+----
+
+The keywords for those need to be surrounded by curly braces:
+----
+ [source,{ini}]
+ ----
+ // some code
+ ----
+----
+
+=== Pin listings
+
+
+.Compact Form
+
+----
+* **comp.group.**_N_**.pin** '(type, direction)' - Functional description
+----
+
+Example:
+====
+* **halui.joint.**_N_**.select** '(bit, in)' - Pin for selecting joint N
+* **halui.joint.**_N_**.is-selected** '(bit, out)' - Status pin that joint N is selected
+* **halui.joint.**_N_**.has-fault** '(bit, out)' - Status pin telling that joint N has a fault
+====
+Where _N_ is the instance number of the component. If it's a componemt/module that can only be loaded once, a zero could/should replace the _N_:
+
+====
+* *iocontrol.0.coolant-flood* '(Bit, Out)' TRUE when flood coolant is requested
+
+* *iocontrol.0.coolant-mist* '(Bit, Out)' TRUE when mist coolant is requested
+
+* *iocontrol.0.emc-enable-in* '(Bit, In)' Should be driven FALSE when an external
+  estop condition exists.
+====
+
+[NOTE]
+(Bit, In) or (bit, in) ?
+
+.Expanded (extended, enlarged?) Form, variant 1
+
+----
+* **comp.group.**_N_**.pin**:: '(type, direction)' - Functional description
+----
+
+Example:
+====
+**pid.**_N_**.Pgain**:: '(float, in)' - Proportional gain. Results in a contribution to the output that is the error multiplied by Pgain.
+
+**pid.**_N_**.Igain**:: '(float, in)' - Integral gain. Results in a contribution to the output that is the integral of the error multiplied by Igain. For example an error of 0.02 that lasted 10 seconds would result in an integrated error '(errorI)' of 0.2, and if Igain is 20, the integral term would add 4.0 to the output.
+
+**pid.**_N_**.Dgain**:: '(float, in)'- Derivative gain. Results in a contribution to the output that is the rate of change (derivative) of the error multiplied by Dgain. For example an error that changed from 0.02 to 0.03 over 0.2 seconds would result in an error derivative (errorD) of of 0.05, and if Dgain is 5, the derivative term would add 0.25 to the output.
+====
+
+.Expanded (extended, enlarged?) Form, variant 2
+----
+* **comp.group.**_N_**.pin** '(type, direction)':: Functional description
+----
+
+Example:
+====
+**pid.**_N_**.Pgain** '(float, in)':: Proportional gain. Results in a contribution to the output that is the error multiplied by Pgain.
+
+**pid.**_N_**.Igain** '(float, in)':: Integral gain. Results in a contribution to the output that is the integral of the error multiplied by Igain. For example an error of 0.02 that lasted 10 seconds would result in an integrated error (`errorI`) of 0.2, and if Igain is 20, the integral term would add 4.0 to the output.
+
+**pid.**_N_**.Dgain** '(float, in)':: Derivative gain. Results in a contribution to the output that is the rate of change (derivative) of the error multiplied by Dgain. For example an error that changed from 0.02 to 0.03 over 0.2 seconds would result in an error derivative (errorD) of of 0.05, and if Dgain is 5, the derivative term would add 0.25 to the output.
```

Comment:
Everything left of the :: is already bold-faced, so the extra ** are not needed. Also, why not have the N both italic and bold?  This would simplify the world a bit._

Reply:
>Everything left of the :: is already bold-faced, so the extra ** are not needed. 

Not with the LinuxCNC stylesheets:
![grafik](https://user-images.githubusercontent.com/67957916/188284799-ae560d80-b935-4357-9776-390441b0f73e.png)

But I wanted this for discussion :). The style is derived from how most of the man pages are formatted (with the italic but non bold N). But it's maybe to much formatting overhead. We can play a bit around with some combinations of bold and non bold and italic and so on what looks best.